### ISSUE TITLE
Unable to convert a role resource

### ISSUE THREAD

**Issue by jeunii** [DontShaveTheYak/cf2tf#204]
Currently using version `0.6.0` on MacOS

I have a simple CF file

```
---
AWSTemplateFormatVersion: 2010-09-09
Description: A basic template

Resources:
  TransferS3AccessRole:
    Type: AWS::IAM::Role
    Properties:
      Description: IAM role used by Transfer to give access to S3 bucket after user is authenticated
      AssumeRolePolicyDocument:
        Version: '2012-10-17'
        Statement:
          - Effect: Allow
            Principal:
              Service:
                - transfer.amazonaws.com
            Action:
              - sts:AssumeRole
 ```

When i try to convert it, Im getting the error

```
➜ cf2tf file.yml
// Converting file.yml to Terraform!
// Existing Terraform src code found at /var/folders/px/c7p93jyj15sd3mf28gz0p95m0000gp/T/terraform_src.
Traceback (most recent call last):
  File "/opt/homebrew/lib/python3.10/site-packages/cf2tf/terraform/doc_file.py", line 19, in parse_attributes
    attributes = parse_section("Attributes Reference", file)
  File "/opt/homebrew/lib/python3.10/site-packages/cf2tf/terraform/doc_file.py", line 37, in parse_section
    section_location = find_section(name, file)
  File "/opt/homebrew/lib/python3.10/site-packages/cf2tf/terraform/doc_file.py", line 117, in find_section
    raise Exception()
Exception

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/homebrew/bin/cf2tf", line 8, in <module>
    sys.exit(cli())
  File "/opt/homebrew/lib/python3.10/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "/opt/homebrew/lib/python3.10/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "/opt/homebrew/lib/python3.10/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/homebrew/lib/python3.10/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/opt/homebrew/lib/python3.10/site-packages/cf2tf/app.py", line 44, in cli
    config = TemplateConverter(tmpl_path.stem, cf_template, search_manger).convert()
  File "/opt/homebrew/lib/python3.10/site-packages/cf2tf/convert.py", line 92, in convert
    tf_resources = self.convert_to_tf(self.manifest)
  File "/opt/homebrew/lib/python3.10/site-packages/cf2tf/convert.py", line 139, in convert_to_tf
    tf_resources.extend(converter(resources))
  File "/opt/homebrew/lib/python3.10/site-packages/cf2tf/convert.py", line 300, in convert_resources
    valid_arguments, valid_attributes = doc_file.parse_attributes(docs_path)
  File "/opt/homebrew/lib/python3.10/site-packages/cf2tf/terraform/doc_file.py", line 21, in parse_attributes
    raise Exception(f"Unable to find attributes in {file.name}") from e
Exception: Unable to find attributes in /var/folders/px/c7p93jyj15sd3mf28gz0p95m0000gp/T/terraform_src/website/docs/r/iam_role.html.markdown
```

Any idea why this is ?

**Comment by shadycuz** [comment]
I will take a look, I see several issues reporting a similar error, might be a bug in the most recent version of `cf2tf`.

**Comment by shadycuz** [comment]
@jeunii Terraform decided to rename `Attributes` to `Attribute` in https://github.com/hashicorp/terraform-provider-aws/pull/32555 which broke our ability to parse correctly. This has been fixed in [v0.6.1](https://github.com/DontShaveTheYak/cf2tf/releases/tag/0.6.1).
